src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -1,35 +1,79 @@
+import { Box, Link, Skeleton, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { fetchPerson } from '../../../api/cristinApi';
 import { fetchRegistration } from '../../../api/registrationApi';
-import { Registration, RegistrationStatus } from '../../../types/registration.types';
+import { ProfilePicture } from '../../../components/ProfilePicture';
+import { PublicationNote, Registration, RegistrationStatus } from '../../../types/registration.types';
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { getFullCristinName } from '../../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
 
 interface DeletedRegistrationInformationProps {
   registration: Registration;
+  unpublishingNote: PublicationNote;
 }
 
-export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+function extractId(createdBy: string) {
+  return '/cristin/person/' + createdBy.split('@')[0];
+}
+
+export const DeletedRegistrationInformation = ({
+  registration,
+  unpublishingNote,
+}: DeletedRegistrationInformationProps) => {
   const { t } = useTranslation();
+
+  const duplicateRegistrationIdentifier = getIdentifierFromId(registration.duplicateOf ?? '');
+
   const duplicateRegistrationQuery = useQuery({
-    enabled: !!registration.duplicateOf,
-    queryKey: ['registration', registration.duplicateOf],
-    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    enabled: !!duplicateRegistrationIdentifier,
+    queryKey: ['registration', duplicateRegistrationIdentifier],
+    queryFn: () => fetchRegistration(duplicateRegistrationIdentifier),
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
+  const cristinIdentifier = unpublishingNote.createdBy ? extractId(unpublishingNote.createdBy) : null;
+  const personQuery = useQuery({
+    queryKey: ['cristinPerson', cristinIdentifier],
+    queryFn: () => fetchPerson(cristinIdentifier ?? ''),
+    enabled: !!cristinIdentifier,
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const person = personQuery.data;
+
   const duplicateRegistrationTitle = duplicateRegistrationQuery.data?.entityDescription?.mainTitle;
-  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
 
   return (
     <StyledStatusMessageBox
       sx={{
         bgcolor: 'publishingRequest.main',
       }}>
-      {(registration.status === RegistrationStatus.Unpublished ||
-        registration.status === RegistrationStatus.Deleted) && (
+      {registration.status === RegistrationStatus.Deleted ? (
         <Typography>{t(`registration.status.${registration.status}`)}</Typography>
+      ) : (
+        <Typography>{t(`registration.status.UNPUBLISHED`)}</Typography>
       )}
```

Comment:
Denne virker litt snedig, vi skal vel alltid vise status til registreringen her, eller?
Denne vil automatisk vise tekst til DELETED om Deleted, og UNPUBLISHED om Unpublished
```suggestion
      <Typography>{t(`registration.status.${registration.status}`)}</Typography>
```